app/views/devise/confirmations/new.html.haml

```diff
@@ -0,0 +1,10 @@
+%h2 Resend confirmation instructions
+= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f|
+  = devise_error_messages!
+  .field
+    = f.label :email
+    %br/
```

Comment:
When you convert `.erb` files to `.haml` remember to remove unnecessary trailing `/`.
The `<br>` tag was not converted properly, since the converter expected `<br>` instead of `<br/>`. Thus, it was converted to `%br/`.